nix/cardano/packages/haskell.nix

```diff
@@ -65,6 +66,7 @@ in
 
       withHoogle = true;
     };
+    inherit evalSystem;
```

Comment:
What does evalSystem default to without this change?

Reply:
`nixpkgs.system`, see https://github.com/input-output-hk/cardano-world/pull/42/files#diff-0ff9c82a12f0e0902d2fb4b5a9980b36a3f106eda195338e5af825f60efad701R23